addon/components/uni-input-price.js

```diff
@@ -54,11 +56,15 @@ export default Component.extend({
     },
 
     onFocusIn() {
-      $('input').focus(function() {
-        $(this).parent('.uni-input-price').addClass('uni-input-price--focused');
-      }).blur(function() {
-        $(this).parent('.uni-input-price').removeClass('uni-input-price--focused');
-      });
+      document.getElementById(this.get('elementId')).classList.add(FOCUSED_CLASS);
```

Comment:
doesn't seem to be equivalent to what we had before 🤔

Reply:
Discussed offline 😄